commit 4e929764d0add05670185d7f8ca19d374dce2e1f (refs/changes/06/106/1)
Author: Kris Shannon <kris@shannon.id.au>
Date:   2016-07-19T14:18:51-07:00 (3 years, 3 months ago)
    
    joyent/node-triton-tags#1 Port range error message doesn't match the code

diff --git a/lib/index.js b/lib/index.js
index a6f543f..d09c384 100644
--- a/lib/index.js
+++ b/lib/index.js
@@ -83,7 +83,7 @@ var validatorFromKey = {
                 if (isNaN(svc.port) || svc.port < 1 || svc.port > 65535) {
                     return format(
                         'invalid "triton.cns.services" tag: service port ' +
-                        'number for %s must be within the range 0 - 65535',
+                        'number for %s must be within the range 1 - 65535',
                         svc.name);
                 }
             }
